README.md

```diff
@@ -51,12 +52,25 @@ Registered entry points for aiida.calculations:
 * mlip.train
 * mlip.descriptors
 ```
+## Prerequisites
```

Comment:
I think this is maybe a bit confusing to have after `Installation`, since it implies these things are required before anything else.

Perhaps it makes sense to have a overall heading of `Getting Started`, and subheadings for `Installation` (same as you have - installing the plugin itself) and then another subheading along the lines of `AiiDA configuration`, which I think should very briefly summarise what needs doing (i.e. configuring a profile/computer/code + optional(?) broker/database), keeping the commands more or less as they are though, just with more context.